File: c/2.c
#include <stdio.h>

int main() {
  int oneNum;
  int twoNum;
  int threeNum;

  scanf("%d", &oneNum);
  scanf("%d", &twoNum);
  scanf("%d", &threeNum);
  // add all together
  int sum = oneNum + twoNum + threeNum;
  printf("Your total number is: %d", sum);

  return 0;
}

File: dart/2.dart
import "dart:io";
void main() {
  //2
  // dart is null safe so the ! is incase it is null
  String onest = stdin.readLineSync()!;
  String twond = stdin.readLineSync()!;
  String threerd = stdin.readLineSync()!;
  // add all together
  String total = "${int.parse(onest)+int.parse(twond)+int.parse(threerd)}";
  print("number converted to int is ${total}");
}